catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: osm2pgsql-wrapper
+  description: A wrapper for osm2pgsql supporting job / cronjob functionality for creating and appending data.
+  tags:
+    - wrapper
```

Comment:
I wouldnt use wrapper as tag.... since it is a wrapper for osm2pgsql I would use osm2pgsql as a tag.

and osm2pgsql-wrapper is not for ingestion but for syncing osm changes